.travis.sh

```diff
@@ -12,7 +12,12 @@ set -e # Error build immediately if install script exits with non-zero
 # if this is a docker job, run in the container instead; but if not just run it here.
 if [ -n "$DOCKER_IMAGE" ]; then
     ./util/dockerize.sh ./.ci.sh
+elif [ "${TRAVIS_OS_NAME}" = "windows" ]; then
+    choco install wsl-ubuntu-1804
```

Comment:
This probably needs

```suggestion
    choco install -y wsl-ubuntu-1804
```

to be reliable. otherwise it might hang waiting for user input.

Reply:
Good idea!